scheduled-jobs/build/oc-mirror-quay-dev/Jenkinsfile

```diff
@@ -1,68 +1,58 @@
+node {
+    checkout scm
 
+    def commonlib = load( "pipeline-scripts/commonlib.groovy")
+    def buildlib = load("pipeline-scripts/buildlib.groovy")
 
-properties(
-    [
-        buildDiscarder(
-            logRotator(
-                artifactDaysToKeepStr: '',
-                artifactNumToKeepStr: '',
-                daysToKeepStr: '',
-                numToKeepStr: '360'
-            )
-        ),
+    properties(
         [
-            $class : 'ParametersDefinitionProperty',
-            parameterDefinitions: [
-                [
-                    name: 'MOCK',
-                    description: 'Mock run to pickup new Jenkins parameters?',
-                    $class: 'hudson.model.BooleanParameterDefinition',
-                    defaultValue: false,
-                ],
-                [
-                    name: 'BUILD_VERSION',
-                    description: 'OCP Version to build',
-                    $class: 'hudson.model.ChoiceParameterDefinition',
-                    choices: "4.0",
-                    defaultValue: '4.0'
-                ],
-                [
-                    name: 'TARGET_NODE',
-                    description: 'Jenkins agent node',
-                    $class: 'hudson.model.StringParameterDefinition',
-                    defaultValue: 'openshift-build-1'
-                ],
-                [
-                    name: 'MAIL_LIST_SUCCESS',
-                    description: 'Success Mailing List',
-                    $class: 'hudson.model.StringParameterDefinition',
-                    defaultValue: [
-                        'aos-team-art@redhat.com',
-                    ].join(',')
-                ],
-                [
-                    name: 'MAIL_LIST_FAILURE',
-                    description: 'Failure Mailing List',
-                    $class: 'hudson.model.StringParameterDefinition',
-                    defaultValue: [
-                        'aos-team-art@redhat.com',
-                    ].join(',')
-                ],
-
-            ]
-        ],
-        disableConcurrentBuilds()
-    ]
-)
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '360'
+                )
+            ),
+            [
+                $class : 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
```

Comment:
might as well:
```suggestion
                    commonlib.mockParam(),
```

Reply:
as above. not going to keep updating this